src/Delorean/Local/Date.hs

```diff
@@ -72,7 +72,7 @@ import           Text.Printf
 newtype Year =
   Year {
     unYear :: Int
-  } deriving (Eq, Show, Ord, Enum, Bounded, Typeable, Data)
+  } deriving (Eq, Ord, Read, Show, Enum, Bounded, Typeable, Data)
```

Comment:
I reordered these because I think Eq/Ord go together, and so does Read/Show, if that's a problem I can change it back.